requirements.txt

```diff
@@ -2,7 +2,6 @@ torch >= 2.0.1        # all
 jupyterlab >= 4.0     # all
 tiktoken >= 0.5.1     # ch02; ch04; ch05
 matplotlib >= 3.7.1   # ch04; ch05
-numpy >= 1.24.3       # ch05
```

Comment:
good catch here

Reply:
Actually `numpy` (and `pandas`) req is not needed as will be installed as dependency from other libraries.

Thanks for the review, was the last PR for minor fixes then.

Reply:
Yeah, but with numpy 2.0 it's a bit tricky at the moment since some dependencies don't support it yet

Reply:
Agree - for now, I would keep it like that, yes. 

But once there is a new major version of a library like pytorch that requires numpy 2.x, this library probably won't work as it would break the dependency (even though I don't think this will happen the near future for pytorch 2.x.x versions).